tests/integration/pkg/tests/ociregistry_tests.go

```diff
@@ -52,9 +52,9 @@ var _ = ginkgo.Describe("[Verify oci registry is working properly]", func() {
 	ginkgo.It("POST requests should be denied", func() {
 		responseBody := bytes.NewBuffer(nil)
 
-		// Nginx proxy should return a 403 forbidden error
+		// OCI server proxy should return a 404 not found error for non-GET requests
```

Comment:
404 seems to be default response here since the server is treating any other method as unimplemented or nonexistent. Same would be given if you tried to POST /index for example.

Reply:
Okay, 405 is more accurate status code for method not allowed, but we can handle this in separate issue as the issue is generic.

Reply:
Opened https://github.com/devfile/api/issues/543 for tracking

Reply:
Sounds good, thanks